go/client/getentries.go

```diff
@@ -29,7 +30,7 @@ func (c *LogClient) GetEntries(start, end int64) ([]ct.LogEntry, error) {
 		return nil, errors.New("start should be <= end")
 	}
 	var resp getEntriesResponse
-	err := fetchAndParse(c.httpClient, fmt.Sprintf("%s%s?start=%d&end=%d", c.uri, GetEntriesPath, start, end), &resp)
+	err := fetchAndParse(context.TODO(), c.httpClient, fmt.Sprintf("%s%s?start=%d&end=%d", c.uri, GetEntriesPath, start, end), &resp)
```

Comment:
I like it!  Presumably one should continue from here by propagating the context back through the call stack until you reach a frame where the caller has a context to hand, or can create one, then the TODO goes away?